mbedtls.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@DEST_DIR@
+exec_prefix=${prefix}
+libdir=${prefix}/lib
+includedir=${prefix}/include
+
+Name: mbedtls
+Description: lightweight crypto and SSL/TLS library.
+Version: @MBEDTLS_VERSION@
+
```

Comment:
Why is there a newline here? It's unusual to have this in the pkg-config files I've seen.